Docs/source/running_cpp/parameters.rst

```diff
@@ -210,11 +210,17 @@ Particle initialization
     particles are pushed in a standard way, using the specified pusher.
     (see the parameter ``<species_name>.zinject_plane`` below)
 
-* ``<species_name>.charge`` (`float`)
+* ``<species_name>.species_type`` (`string`) optional (default `unspecified`)
+    Type of physical species, ``"electron"``, ``"positron"``, ``"photon"``, ``"hydrogen"``.
+    Either this or both ``mass`` and ``charge`` have to be specified.
```

Comment:
For the moment I think that these options for the type could be enough... However, in the future, it could be nice to add at least the elements most often used in simulations (e.g. `helium`, `carbon`, `aluminium`, `titanium`, `silicon`, `gold`). With **a lot of patience** we could even add all the elements in the periodic table, the most common isotopes and even other particles (e.g. `muons`), but I am not suggesting to do that right now!

Reply:
That also points to how we link it with ionization, which is a good point.